config/initializers/hyrax.rb

```diff
@@ -231,10 +246,20 @@
   # Increase the number of Asset members that are displayed on the Asset's
   # #show page.
   config.show_work_item_rows = 100
+
+  config.nested_relationship_reindexer = ->(id:, extent:) { QueuedNestingIndexer.reindex_relationships(id: id, extent: extent) }
+
 end
 
 Date::DATE_FORMATS[:standard] = "%m/%d/%Y"
 
 Qa::Authorities::Local.register_subauthority('subjects', 'Qa::Authorities::Local::TableBasedAuthority')
 Qa::Authorities::Local.register_subauthority('languages', 'Qa::Authorities::Local::TableBasedAuthority')
 Qa::Authorities::Local.register_subauthority('genres', 'Qa::Authorities::Local::TableBasedAuthority')
+
+# set bulkrax default work type to first curation_concern if it isn't already set
+if ENV['SETTINGS__BULKRAX__ENABLED'] == 'true'
+  if Bulkrax.default_work_type.blank?
+    Bulkrax.default_work_type = Hyrax.config.curation_concerns.first.to_s
```

Comment:
yes.

Reply:
I think I'd prefer if it was just explicit then, i.e. `Bulkrax.default_work_type = "Asset"`, because I can't tell whether `Bulkrax.default_work_type` has already been set, and if not, then I'm not sure how we are guaranteed that `Hyrax.config.curation_concerns.first` is `Asset`.

Reply:
👍 already changed it